### ISSUE TITLE
Redux

### ISSUE THREAD

**Issue by viktorovich87** [asd95/tt-adminui#9]
Как обстоят дела с redux?

вопрос обобщенный, потому как он не был использован в проекте. как бы требований не было, но просто интересно узнать какое общение у вас с redux?

**Comment by asd95** [comment]
Да, думал использовать redux. Но не хотел его применять ради двух- трех функций. Больше бы время заняло. 
У меня есть проект на гитхабе. Там я применил его. По нему можно судить. Супер тонкости не знаю. 
Знаю принцип его работы, как делать ассинхронные запросы. 
https://github.com/asd95/mania-mp вот ссылка. Там более менее понятно с чем я имел дело. Дальше не углублялся.
Мне сказали на одном собеседовании что он умирает) типа лучше изучай mobx

**Comment by viktorovich87** [comment]
ленин жил, ленин жив , ленин будет жить..
редакс очень классная библиотека, которая не скоро сместится другими техзнологиями..
понятно, спасибо)

**Comment by asd95** [comment]
Я почти так же им ответил, только в более деликатной форме ибо они опытнее меня, но я смотрю более крутых ребят которые говорили обратное.
